build/release.mk

```diff
@@ -82,14 +82,14 @@ endif
 	gpg --import build/plugin-production-public-key.gpg
 
 	@# Download prepackaged plugins
-	@for plugin_package in $(PLUGIN_PACKAGES) ; do \
-		curl -f -o $(DIST_PATH)/prepackaged_plugins/$$plugin_package.tar.gz -L https://plugins-store.test.mattermost.com/release/$$plugin_package.tar.gz; \
-		curl -f -o $(DIST_PATH)/prepackaged_plugins/$$plugin_package.tar.gz.sig -L https://plugins-store.test.mattermost.com/release/$$plugin_package.tar.gz.sig; \
-		gpg --verify $(DIST_PATH)/prepackaged_plugins/$$plugin_package.tar.gz.sig $(DIST_PATH)/prepackaged_plugins/$$plugin_package.tar.gz; \
-		if [ $$? -ne 0 ]; then \
-			echo "Failed to verify" $$plugin_package.tar.gz"|"$$plugin_package.tar.gz.sig; \
-			exit 1; \
-		fi; \
+	mkdir -p tmpprepackaged
+	@cd tmpprepackaged && for plugin_package in $(PLUGIN_PACKAGES) ; do \
+		curl -f -O -L https://plugins-store.test.mattermost.com/release/$$plugin_package-osx-amd64.tar.gz; \
```

Comment:
Worth structuring this using the same `ARCH="osx-amd64";` assignment and then duplicating the lines for clarity?